TECH.md

```diff
@@ -0,0 +1,4 @@
+OpenAPI Space is a Python API built with Flask and Connexion
+ - Flask is a microframework for building Python based web servers. We concluded that it fits well into building a backend which’s main function is to provide a REST AP
```

Comment:
`which's` -> `where the`?
`REST AP` -> `REST API`

Reply:
Whoops, thanks for the corrections.